### PR TITLE
Update Text-Korean-1042.txt

### PR DESCRIPTION
**Pull request by VenusGirl**

### CODE REVIEW

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 1]
```diff
@@ -2762,100 +3956,103 @@ Sandboxie 제어판을 엽니다
...
 .
 
-8107;txt;01
-다음 프로그램을 종료한 뒤 확인 버튼을 눌러 진행하십시오.
-취소 버튼을 누르면 설치를 중단합니다.
+8106;txt;02
+설치를 계속하려면 다음 프로그램을 닫아야 합니다.
+이러한 프로그램을 닫고 계속하려면 확인을 클릭하십시오. 설치를 중단하려면 취소를 클릭하십시오.
 .
+Footer
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:4058]:
```suggestion

```

**Review by isaak654 — CHANGES_REQUESTED** [review]

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 2]
```diff
@@ -105,67 +105,75 @@ SBIE1153 Sandboxie 초기화 실패. 모든 프로그램을 닫고 Sandboxie를
...
 # 1217;pop;err;01
-# SBIE1217 현재 프로세스의 상위 프로세스를 찾을 수 없습니다
+# SBIE1217 상위 프로세스를 찾을 수 없습니다.
+# .
+
+# 1221;pop;err;01
+# SBIE1221 작업을 생성할 수 없습니다: %2
 # .
 
 1222;pop;err;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:155]:
```suggestion
1222;pop;err;02
```

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 3]
```diff
@@ -175,35 +183,75 @@ SBIE1242 감시 버퍼가 사용 한계를 초과했습니다
...
+# 1304;pop;inf;01
+# SBIE1304 '%2' 프로세스에 의해 시뮬레이션된 키보드 또는 마우스 입력을 차단했습니다
+# .
 
 1306;pop;inf;01
-SBIE1306 Sandboxie 드라이버 (SbieDrv)를 내릴 수 없음
+SBIE1306 샌드박스 드라이버(SbieDrv)를 지금 내릴 수 없습니다
 .
 
 1307;pop;inf;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:205]:
```suggestion
1307;pop;inf;02
```

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 4]
```diff
@@ -175,35 +183,75 @@ SBIE1242 감시 버퍼가 사용 한계를 초과했습니다
...
-SBIE1307 제한 설정에 따라 프로세스 는 인터넷에 접근할 수 없습니다 - %2
+SBIE1307 제한으로 인해 프로그램이 인터넷에 접근할 수 없습니다 - %2
 .
 
 1308;pop;inf;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:209]:
```suggestion
1308;pop;inf;02
```

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 5]
```diff
@@ -953,21 +1236,37 @@ Sandbox(&B)
 .
 
 3453;txt;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:1238]:
```suggestion
3453;txt;02
```

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 6]
```diff
@@ -1464,114 +1838,119 @@ Sandbox를 만들고 저장할 드라이브를 선택.
...
 3686;txt;01
 Sandboxie 제어판 시작
 .
 
 3687;txt;01
-Windows를 시작할 때
+Windows가 시작될 때
 .
 
 3688;txt;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:1860]:
```suggestion
3688;txt;02
```

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 7]
```diff
@@ -1753,35 +2174,43 @@ Sandbox 설정 - %2
...
 3804;txt;01
-다른 페이지로 이동할 때 변경 사항 적용
+다른 페이지로 전환할 때 변경 사항을 적용합니다.
 .
 
 3805;txt;01
 모든 프로그램
 .
 
 3806;txt;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:2192]:
```suggestion
3806;txt;02
```

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 8]
```diff
@@ -2662,95 +3856,95 @@ Sandboxie를 지원해 주셔서 감사합니다!
...
-Sanboxie의 시스템 드라이버를 설치하고 활성화하기 위해 다음 단추를 클릭하세요. 이 드라이버는 Sandboxie 프로그램의 핵심입니다.
+Sanboxie 시스템 수준 드라이버를 설치하고 활성화하려면 다음을 누르십시오. 이 드라이버는 Sanboxie 응용 프로그램의 핵심입니다.
 .
 
 8013;ins;01
-시스템 드라이버의 설치를 막는 시스템 보호 도구를 일시적으로 사용 중지해야 할 수도 있습니다.
+실행 중인 시스템 보호 도구를 일시적으로 비활성화해야 할 수 있으므로 시스템 수준 드라이버가 설치되지 않을 수 있습니다.
 .
 
 8014;ins;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:3906]:
```suggestion
8014;ins;02
```

**Review by isaak654 — CHANGES_REQUESTED** [review]
First round of suggested fixes.

**Comment by isaak654** [comment]
#2150

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 9]
```diff
@@ -502,61 +664,53 @@ SBIE9154 드라이버 (SbieDrv)와 서비스 (SbieSvc)의 판번호가 다름
...
-.
+# 9208;evt;err;01
+# SBIE9208 SeRestorePrivilege를 활성화할 수 없습니다
+# .
+
+# 9251;evt;err;01
+# SBIE9251 포트 이벤트를 생성하지 못하였습니다
+# .
 
 9252;evt;err;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:707]:
```suggestion
# 9252;evt;err;01
```

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 10]
```diff
@@ -1793,534 +2222,720 @@ Sandbox 설정 - %2
...
 .
 
 4032;txt;01
 읽기 전용 파일 접근 (ReadFilePath)
 .
 
 4033;txt;01
-다음 파일과 폴더는 이 Sandbox에서 실행된 프로그램이 바꿀 수 없습니다.
+다음 파일 및 폴더는 이 샌드박스에서 실행 중인 프로그램으로 수정할 수 없습니다.
+..
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:2807]:
```suggestion
.
```

**Review by isaak654 — CHANGES_REQUESTED** [review]
Second round of suggested fixes.

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 11]
```diff
@@ -502,61 +664,53 @@ SBIE9154 드라이버 (SbieDrv)와 서비스 (SbieSvc)의 판번호가 다름
...
-SBIE9304 섹션을 만들지 못하였습니다 (장치 ID 목록)
-.
+# 9252;evt;err;01
+# SBIE9252 포트를 생성하지 못하였습니다
+# .
 
-9305;evt;err;01
-SBIE9305 보호 모드로 실행중인 모든 프로그램을 닫으십시오
-.
+9253;evt;err;01
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:711]:
```suggestion
# 9253;evt;err;01
```

**Review by isaak654 — CHANGES_REQUESTED** [review]
Third round

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 12]
```diff
@@ -175,35 +183,75 @@ SBIE1242 감시 버퍼가 사용 한계를 초과했습니다
...
+#.
+
+1312;pop;inf;01
+SBIE1312 샌드박스에서 16비트 DOS 프로그램을 시작하라는 요청을 차단했습니다
 .
 
-1307;pop;inf;01
-SBIE1307 제한 설정에 따라 프로세스 는 인터넷에 접근할 수 없습니다 - %2
+1313;pop;inf;01
+SBIE1313 SBIE1313 프로세스 '%2'에 의해 직접 디스크 접근이 차단되었습니다
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:230]:
```suggestion
SBIE1313 프로세스 '%2'에 의해 직접 디스크 접근이 차단되었습니다
```

**Review by isaak654 — CHANGES_REQUESTED** [review]
I suspect there could be more duplicates, I recommend you to review the translation one last time.

File: Sandboxie/msgs/Text-Korean-1042.txt [hunk 13]
```diff
@@ -502,61 +664,53 @@ SBIE9154 드라이버 (SbieDrv)와 서비스 (SbieSvc)의 판번호가 다름
...
-.
-
-9202;evt;err;01
-SBIE9202 Token 오류입니다
-.
-
-9203;evt;err;01
-SBIE9203 Token 오류입니다
+9234;evt;err;01
+SBIE9234서비스 시작 오류 %2
```
Comment (isaak654) [inline Sandboxie/msgs/Text-Korean-1042.txt:668]: Missing space:
```suggestion
SBIE9234 서비스 시작 오류 %2
```

**Review by isaak654 — CHANGES_REQUESTED** [review]

**Comment by isaak654** [comment]
Did you perform a final check?

**Comment by isaak654** [comment]
> Did you perform a final check?

@VenusGirl I am waiting for written confirmation, thank you.

**Comment by VenusGirl** [comment]
Yes

**Review by isaak654 — APPROVED** [review]

**Pull request merged**